test/e2e/apimachinery/flowcontrol.go

```diff
@@ -274,9 +275,14 @@ func createPriorityLevel(ctx context.Context, f *framework.Framework, priorityLe
 }
 
 func getPriorityLevelNominalConcurrency(ctx context.Context, c clientset.Interface, priorityLevelName string) (int32, error) {
-	resp, err := c.CoreV1().RESTClient().Get().RequestURI("/metrics").DoRaw(ctx)
+	req := c.CoreV1().RESTClient().Get()
+	reqURL := req.URL()
+	reqPathOrig := reqURL.EscapedPath()
+	reqPathMetrics := strings.TrimSuffix(reqPathOrig, "api/v1") + "metrics"
```

Comment:
Let's add an explicit comment why you're doing those tricks - it didn't seem intuitive before reading the corresponding bug.

[Trimming api/v1 seems like it can break in the future, but I don't have a better way of doing it, so let's leave it].

Reply:
Thanks, I was indeed wondering if there is a better way to do it.

The force-push to aa0e28c870a adds a comment.

Reply:
`c.CoreV1().RESTClient().Get().AbsPath("/metrics").DoRaw(ctx)` would have been a cleaner way to do this. AbsPath honors any base prefix configured for the server, while overriding all the API-path portions of the request.

as a demonstration:

```go
	fmt.Println("no server prefix, RequestURI:", clientset.NewForConfigOrDie(config).CoreV1().RESTClient().Get().RequestURI("/metrics").URL())
	config.Host += "/base"
	fmt.Println("server prefix, RequestURI:", clientset.NewForConfigOrDie(config).CoreV1().RESTClient().Get().RequestURI("/metrics").URL())
	fmt.Println("server prefix, AbsPath:", clientset.NewForConfigOrDie(config).CoreV1().RESTClient().Get().AbsPath("/metrics").URL())
```
produces:
```sh
no server prefix, RequestURI: https://127.0.0.1:51840/metrics
server prefix, RequestURI: https://127.0.0.1:51840/metrics
server prefix, AbsPath: https://127.0.0.1:51840/base/metrics
```